internal/xds/translator/testdata/out/xds-ir/http-route-timeout.routes.yaml

```diff
@@ -14,6 +14,25 @@
       name: first-route
       route:
         cluster: first-route-dest
+        idleTimeout: 3600s
```

Comment:
nit: would be ideal if this wasn't set

Reply:
Two choices:

a) 1hr default set on the route only when the request timeout is passed on the route (as currently implemented)

b) Set `stream_idle_timeout` in the connection manager to 1hr (vs implicit default of 300s), only configuring per-route idle timeout if >1hr or 0

thoughts?

Reply:
Prefer b